openscm/adapters/dice.py

```diff
@@ -0,0 +1,303 @@
+            "c4": (0.00487, ""),
+            # Forcings of equilibrium CO2 doubling (Wm-2)
+            "fco22x": (3.8, "W/m^2"),
+            # 2010 forcings of non-CO2 GHG (Wm-2)
+            "fex0": (0.25, "W/m^2"),
+            # 2100 forcings of non-CO2 GHG (Wm-2)
+            "fex1": (0.7, "W/m^2"),
+            # Equilibrium temp impact (°C per doubling CO2)
+            "t2xco2": (2.9, "°C"),
+        }
+
+        for n, v in model_parameter_defaults.items():
+            default, unit = v
+            setattr(
+                self,
+                "_{}".format(n),
+                self._parameterset.get_scalar_view(("DICE", n), (), unit).get(default),
+            )
+
+        # Carbon cycle transition matrix
+        self._b11 = 1 - self._b12
+        self._b21 = self._b12 * self._M_atm_eq / self._M_u_eq
+        self._b22 = 1 - self._b21 - self._b23
+        self._b32 = self._b23 * self._M_u_eq / self._M_l_eq
+        self._b33 = 1 - self._b32
+
+        # Total CO2 emissions (GtCO2 per year)
+        self._E = self._parameterset.get_writable_timeseries_view(
+            ("Emissions", "CO2"), (), "GtCO2/a", self._start_time, self._period_length
+        )
+
+        # Concentration in atmosphere (GtC)
+        self._M_atm = self._parameterset.get_writable_timeseries_view(
+            ("Concentration", "Atmosphere"),
+            (),
+            "GtC",
+            self._start_time,
+            self._period_length,
+        )
+        self._M_atm.set(0, self._M_atm0)
+
+        # Carbon concentration increase in lower oceans (GtC from 1750)
+        self._M_l = self._parameterset.get_writable_timeseries_view(
+            ("Concentration", "Ocean", "lower"),
+            (),
+            "GtC",
+            self._start_time,
+            self._period_length,
+        )
+        self._M_l.set(0, self._M_l0)
+
+        # Carbon concentration increase in shallow oceans (GtC from 1750)
+        self._M_u = self._parameterset.get_writable_timeseries_view(
+            ("Concentration", "Ocean", "shallow"),
+            (),
+            "GtC",
+            self._start_time,
+            self._period_length,
+        )
+        self._M_u.set(0, self._M_u0)
+
+        # Increase in temperature of lower oceans (°C from 1900)
+        self._T_ocean = self._parameterset.get_writable_timeseries_view(
+            ("Temperature Increase", "Ocean", "lower"),
+            (),
+            "°C",
+            self._start_time,
+            self._period_length,
+        )
+        self._T_ocean.set(0, self._T_ocean0)
+
+        # Increase in radiative forcing (watts per m2 from 1900)
+        self._force = self._parameterset.get_writable_timeseries_view(
+            ("Radiative forcing",), (), "W/m^2", self._start_time, self._period_length
+        )
+
+        # Increase temperature of atmosphere (°C from 1900)
+        self._T_atm = self._parameterset.get_writable_timeseries_view(
+            ("Temperature Increase", "Atmosphere"),
```

Comment:
```suggestion
            ("Surface Temperature Anomaly"),
```

There's almost no way this is meant to be anything other than surface temperature.

Reply:
Not a fan of "anomaly", see page 27 in https://coast.noaa.gov/data/digitalcoast/pdf/psychology-climate-change-communication.pdf

"Surface Temperature Increase"?

Reply:
Will change these according to #22 once we have that

Reply:
ok but our audience here is scientists no (I can't imagine using the source as a communication tool...?)

Reply:
> ok but our audience here is scientists no (I can't imagine using the source as a communication tool...?)

What do you mean by that?
All adapters should use the standard names which we are defining in #22, so let's do that first

Reply:
(maybe this belongs in #22) @rgieseke and I are disagreeing about what the standard name for 'Surface Temperature Change' should be. @rgieseke links https://coast.noaa.gov/data/digitalcoast/pdf/psychology-climate-change-communication.pdf and hence against using 'anomaly', I'd argue that our audience (also referenced in Section 1 or something of that doc) is not the public but rather scientists so let's use the word that means what we mean and stick to 'anomaly'.

Reply:
ah, ok ;)